### DISCUSSION TITLE
Style : Info modules once wallet connected

### DISCUSSION THREAD

**Discussion by Luxus-xyz** [rainbow-me/rainbowkit#525] [Q&A] [answered]
Hey, is there a possibilities once your wallet connected, that we could put boxes (Chain selectors and wallet info) in flex-direction : column ?
![image](https://user-images.githubusercontent.com/95345084/175059031-ed377df8-deef-4ac0-8b6e-53e5ea462c94.png)

**Comment by saihaj** [comment, marked as answer]
you can customize however you want https://www.rainbowkit.com/docs/custom-connect-button
